### ISSUE TITLE
add progress spinner for generate alt text action

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#231901]
cc @tyriar, who had this idea

**Comment by justschen** [comment]
instead of a progress spinner, we show a progress bar on long code actions.  example:
![Image](https://github.com/user-attachments/assets/59518c7f-caa7-4634-8c00-e05f7699075a)


to verify:
1. a code action that takes a long time (example: if you delete all your imports -> quick fix -> add all missing imports. or download our new Vision for Copilot Preview extension, pop in an api key, and try the `Alt Text Generation` quick fix 😄 )
2. see that there is an editor progress bar!
